packages/frontend-2/lib/viewer/composables/setup/selection.ts

```diff
@@ -31,6 +31,15 @@ function useSelectionStateSync() {
     state.ui.selectedObjects.value = objs
   }
 
+  watch(state.ui.selectedObjects, (newVal) => {
```

Comment:
make sure to test this doesnt cause an infinite loop: selectedObjects updates viewer updates selectedObjects updates viewer etc. etc.

in some cases for some sync we used to disable watchers while making the mutation, to avoid this.